ngraph/test/backend/recurrent_cells.in.cpp

```diff
@@ -403,6 +402,13 @@ NGRAPH_TEST(${BACKEND_NAME}, rnn_cell_sigmoid_activation_function) {
 }
 
 NGRAPH_TEST(${BACKEND_NAME}, gru_cell_bias_clip) {
+    /* @todo 48230
+     * Some slight accuracy diviations after recent MatMul changes
+     * (for CPU plugin gru_cell with clip decomposes into MatMuls)
+     * Fix or increase threashold */
+    if (std::string("${BACKEND_NAME}") == std::string("IE_CPU"))
+        return;
```

Comment:
@echuraev
Are you ok with the change?
The problem is that after using new MatMul primitive in CPU plugin we got some accuracy deviations in comparison with the previous implementation.
Those deviations are inside a threshold in case of Openvino functional tests.
But ngraph backend tests check for exact match
Let me know if there is better way to deal with this issue.

Reply:
Please use this file: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/runtime/ie/unit_test.manifest
To disable specific unit tests

Reply:
Thanks,
Done